homeassistant/components/aws/notify.py

```diff
@@ -32,10 +32,10 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-async def get_available_regions(hass, service):
+async def get_available_regions(hass: HomeAssistant, service: str) -> list[str]:
     """Get available regions for a service."""
     session = AioSession()
-    return await session.get_available_regions(service)
+    return await hass.async_add_executor_job(session.get_available_regions, service)
```

Comment:
We can't run a coro in the executor.

This probably needs to be fixed in the library itself or we can work around it by calling whatever its calling under the hood if its initializing / loading a cache from disk before this function so its not called in the event loop when this function does get called.